utils/helpers/managerHelper.ts

```diff
@@ -40,6 +41,9 @@ const MACOStrategyManagerV2 = artifacts.require('MACOStrategyManagerV2');
 const MovingAverageToAssetPriceCrossoverTrigger = artifacts.require(
   'MovingAverageToAssetPriceCrossoverTrigger'
 );
+const RSIMidlineCrossTrigger = artifacts.require(
```

Comment:
nit: But this probably doesn't need to be on multiple lines. It's done like that above because the names are too long to squeeze it all in one line with the linter.